index.js

```diff
@@ -192,11 +192,17 @@ const checkLowerCase = (str) => {
 function validateForm(e) {
```

Comment:
No data is stored when some data is fed in the form fields, and data does not remain in browser after the page is refreshed. Kindly ensure that:
- [ ] Data entered into the form fields are persisted to the local storage on the fly.
- [ ] Persisted data should reflect in the form fields, even when the page is refreshed.

**Entered data not reflecting in local storage:**
![image](https://user-images.githubusercontent.com/12745474/209244664-459ecfbb-30a5-49a3-aec1-daf3b4d45d80.png)

Reply:
hello @rloterh thank you for your review, but I just tested again and the data is store. Can you please check that you are on the branch add-localStorage

Reply:
Hi @SFarida 

**branch add-localStorage:**
![image](https://user-images.githubusercontent.com/12745474/209249678-6350eb5b-779b-4191-95b8-f1b2000a143c.png)

**Live Server (incognito mode):**
![image](https://user-images.githubusercontent.com/12745474/209250090-5228fdee-e07b-4f0d-b265-8b5f274aadee.png)

Reply:
Then this is strange because I have tested again on two different browsers even. 
 
![image](https://user-images.githubusercontent.com/23447217/209250482-e142db35-317a-4b28-97d2-71f55a385fbf.png)

Reply:
I tried with the live server too, 
![image](https://user-images.githubusercontent.com/23447217/209250885-3b6fcb8a-351c-48b4-9daa-8035609a0a50.png)

Reply:
@SFarida 
I this case I assume that perhaps your local code is more updated that the online (Github repository) version

I therefore urge you to kindly **push** your presently working local version to update the online version

Reply:
Kindly checkout my comments on slack.

I also suggest that you work together with your assigned team-mate in finding solutions to this issue.